### ISSUE TITLE
Load the list of people who reacted to a message before mouse hover

### ISSUE THREAD

**Issue by MarcAnt01** [BreeceW/Oxidized#9]
Currently there is a "who reacted" before I see the list of people

**Comment by BreeceW** [comment]
This is not a possibility (loading without user input) because that would require making a separate request for every reaction. What I can try, however, is removing the “Who reacted…” placeholder, though that would remove any indication that it is loading. Let me know if you can think of a more pleasant way to indicate loading before showing who reacted (or even a completely different UI—it doesn’t have to be a tooltip).

**Comment by MarcAnt01** [comment]
Yes, I think that's definitely a good improvement if you cannot get the reactions list in advance

**Comment by BreeceW** [comment]
The way this works today is it loads the people who reacted once the tooltip opens. To mask the delay, you could make it load on pointer over (with a slight delay in case of accidental hovering), that way, the tooltip content would be ready by the time the tooltip opens.
